iac/tf-multienv-cicd-anthos-autopilot/apis.tf

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module "enabled_google_apis" {
+  source  = "terraform-google-modules/project-factory/google//modules/project_services"
+  version = "~> 13.0"
+
+  project_id                  = var.project_id
+  disable_services_on_destroy = false
+
+  activate_apis = [
+    "artifactregistry.googleapis.com",
+    "sourcerepo.googleapis.com",
+    "cloudbuild.googleapis.com",
+    "clouddeploy.googleapis.com",
+    "compute.googleapis.com",
+    "anthos.googleapis.com",
+    "container.googleapis.com",
+    "gkehub.googleapis.com",
+    "gkeconnect.googleapis.com",
+    "anthosconfigmanagement.googleapis.com",
+    "mesh.googleapis.com",
+    "meshconfig.googleapis.com",
+    "meshtelemetry.googleapis.com",
+    "iam.googleapis.com",
+    "sqladmin.googleapis.com"
+  ]
+}
```

Comment:
**Praise:** I like that we're dedicating a separate `apis.tf` file for GCP API enablement. This is clean!

Reply:
🙈 Thanks for the kind words, Nim!